### PR TITLE
Presentation of Multiscale bone modeling research for Related publications section

### PR DESCRIPTION
**Pull request by blaszm**
As discussed via E-Mail, here is a short overview of my research (multiscale bone modeling of cancellous bone) involving the usage of the CoherentStructures package for the "Related publications" section of the documentation. Besides an explanation regarding my use of the package, I included a short introduction to the topic, some important equations and links to open access publications.

Please note that I had some trouble with the make.jl file and could not test the resulting markdown/html file locally. I still hope everything works as intended. 

Please let me know your opinion, I am open for any further suggestions / improvements.

Best, 
Mischa

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -170,6 +173,7 @@ makedocs(
             "Material diffusion barriers" => "generated/diffbarriers.md"
             "Diffusion barriers in turbulence" => "generated/turbulence.md"
             "Life cycles of Loop Current rings" => "generated/lifecycle.md"
+			"Multiscale modeling of bone" => "generated/bone_modeling.md"
```
Comment (dkarrasch) [inline docs/make.jl:176]: Please change tabs to spaces.

File: docs/examples/bone_modeling.jl [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# [Ferrite-periodic-boundary-conditions](https://discourse.julialang.org/t/ferrite-jl-periodic-boundary-conditions/39387).
+#
+# ## Code availability:
+#
+# The code can be found on [Github](https://github.com/blaszm/bone_fe2_c).
+#
+# ## Related publications:
+#
+# 1.	[Multiscale modeling of cancellous bone](https://link.springer.com/article/10.1007/s10237-021-01525-6)
+# 2.	[Inverse modeling using artificial neural networks](https://onlinelibrary.wiley.com/doi/full/10.1002/zamm.202100541)
```
Comment (dkarrasch) [inline docs/examples/bone_modeling.jl:84]: Please add a line break at the end.

**Review by dkarrasch — COMMENTED** [review]
Very well written! Thanks for your contribution. I have only two small style comments. I'll merge the PR once the docs are built and we see that everything works as expected.

**Comment by blaszm** [comment]
Thank you for the kind words and the feedback! Both parts should be fixed. Please let me know when there is anything else.

**Comment by dkarrasch** [comment]
I suspect that the docs don't build because you're not a member of the org. I'll merge this and fix issues afterwards, if necessary. Thanks again!

**Pull request merged**